wordpress-mobile/gutenberg-mobile
# Layout grid - Fix placeholder styles

**Related PRs:**
- https://github.com/Automattic/block-experiments/pull/335

Fixes an issue with the placeholder styles after the recent design changes related to block outlines for selected/unselected states.

To test:

- Open the app using the host apps or [from this iOS build](https://github.com/wordpress-mobile/WordPress-iOS/pull/21217#issuecomment-1660431347)
- Add a Layout grid block
- **Expect** to see the "Select a layout" picker
- Tap on any of the options
- **Expect** to see the picker to be dismissed and the block to show the selected outline

After
-|
<video src="https://github.com/Automattic/block-experiments/assets/4885740/6b8d6d89-6a2a-466c-b778-1e626de6388e" width=200 />|

PR submission checklist:
